README.md

```diff
@@ -4,227 +4,178 @@
 [![CircleCI](https://circleci.com/gh/3scale/apisonator.svg?style=shield)](https://circleci.com/gh/3scale/apisonator)
 [![Maintainability](https://api.codeclimate.com/v1/badges/d2cea8016f0089cb2fd6/maintainability)](https://codeclimate.com/github/3scale/apisonator/maintainability)
 
-This is Red Hat 3scale API Management Platform's Backend.
-
 This software is licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).
 
 See the LICENSE and NOTICE files that should have been provided along with this
 software for details.
 
-## Development environment set up
-
-To learn how to run the project once the environment has been set up, refer to
-the "Running" section.
-
-The first thing you will need is cloning the project:
-> `$ git clone git@github.com:3scale/apisonator.git`
+## Development
 
-Next cd into the directory, `cd apisonator`.
-
-### Containerized environment
-
-#### Prerequisites
-
-* Docker (requires version 1.10.0 or later)
+See the file [DEVELOPMENT](DEVELOPMENT.md)
```

Comment:
maybe move this section way down so that when reading this for the first time you get straight to the explanation of what is this project